jobs/keepalived/templates/bin/action.sh.erb

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+STATE=$1
+iaas_name='<%= link("broker").p('ha.provider.name', nil) %>'
```

Comment:
ha.provider.name === iaas_name ? why is it this way? why not directly pick up `landscape.type`